BAO/Configurations/clcommon__CL_Product__c/12_MonthBusinessCD.json

```diff
@@ -0,0 +1,84 @@
+     "clcommon__Interest_In_Advance__c": false,
+     "clcommon__Pricing_Method__c": null,
+     "clcommon__Product_Abbreviation__c": null,
+     "clcommon__Product_Name__c": "12-Month Business CD",
+     "clcommon__Release_Checksum__c": "fa2900af8a835da0f6c6b4200a35c719129fcbcacfefdcb911fd02e20d745230",
+     "clcommon__Select_If_Letter_Of_Credit__c": false,
+     "clcommon__Status_Tracking_At_Document_File_Level__c": false,
+     "intdm__CS_HELOC_Product_Code__c": null,
+     "intdm__Ceiling_Rate__c": null,
+     "intdm__Days_Convention__c": null,
+     "intdm__Draw_Period_Term_Periods__c": null,
+     "intdm__Draw_Period_Term_Unit__c": null,
+     "intdm__Fixed_Payment_Amount_Draw_Period__c": null,
+     "intdm__Fixed_Payment_Amount_Repayment_Period__c": null,
+     "intdm__Floor_Rate__c": null,
+     "intdm__HELOC__c": false,
+     "intdm__Initial_Interest_Rate_Options__c": null,
+     "intdm__Insider_Loans_Allowed__c": false,
+     "intdm__Interest_Rate_Options__c": null,
+     "intdm__Is_Covered_under_Reg_Z__c": false,
+     "intdm__Payment_Calc_Method_Draw_Period__c": null,
+     "intdm__Payment_Calc_Method_Repayment_Period__c": null,
+     "intdm__Payment_Frequency_Draw_Period__c": null,
+     "intdm__Payment_Frequency_Repayment_Period__c": null,
+     "intdm__Payment_Percentage_Draw_period__c": null,
+     "intdm__Payment_Percentage_Repayment_period__c": null,
+     "intdm__Rate_Revision_Frequency__c": null,
+     "intdm__Relationship_Between_Draw_And_Repayment__c": null,
+     "intdm__Repayment_Period_Term_Periods__c": null,
+     "intdm__Repayment_Period_Term_Unit__c": null,
+     "intdm__UCCC__c": false,
+     "q2adapters__Core_Product_SubType__c": null,
+     "q2adapters__Core_Product_Type__c": null,
+     "solcommon__Recommendation_Rule__r": null,
+     "oao__Active_Product__c": true,
+     "oao__Bank_Account__c": null,
+     "oao__Benefits_Text__c": "Free Mobile and Online Banking<br>FDIC-Insured<br>1.50% APY<br> ",
+     "oao__Benefits_Title__c": "Included Features:",
+     "oao__Configuration_Reference__c": null,
+     "oao__Default__c": false,
+     "oao__Disclosure_Documents__c": null,
+     "oao__Footer__c": "*$4.00 charge per debit in excess of 9 per quarter",
+     "oao__Funding__c": true,
+     "oao__Hidden__c": false,
+     "oao__IsSelected__c": false,
+     "oao__Minimum_Age_Rules__c": null,
+     "oao__Minimum_Funding__c": 100,
+     "oao__Olbenroll__c": true,
+     "oao__Overdraft_Options__c": true,
+     "oao__Product_Id__c": "CD1",
+     "oao__Received_Gro_Products__c": false,
+     "oao__Required__c": false,
+     "oao__Requirements_Text__c": "<table border=\"0\" cellpadding=\"0\" cellspacing=\"0\"><tbody><tr><td colspan=\"1\" rowspan=\"1\">Must be 18 years or older<br>\t\t\tAccount must be opened by an authorized signer</td></tr></tbody></table>",
+     "oao__Requirements_Title__c": "Requirements:",
+     "oao__Term__c": null,
+     "oao__Title__c": null,
+     "Default_Funding__c": 100,
```

Comment:
Add namespace prefix to the newly added fields.
"Default_Funding__c": null,
     "Maximum_Funding__c": 1000,
     "Show_Term__c": false,
     "Product_Term__c": "36",
     "Title__c": "36-Month Business CD"
Do this for all the records below.